### PR TITLE
Create plugin-apprenda.yml

### PR DESCRIPTION
**Pull request by michmike**
# Description

I am uploading the permissions to request upload permissions for a brand new plugin to be hosted for the apprenda cloud platform.

GitHub repo: https://github.com/jenkinsci/apprenda-plugin
Hosting Request: https://issues.jenkins-ci.org/browse/HOSTING-481

I was not 100% i did the following item correctly. i would appreciate a spot check.
- [X ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)

# Submitter checklist for changing permissions

### Always

- [X ] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ X] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [ X] Make sure the file is created in `permissions/` directory
- [X] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [X ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [X ] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ X] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [X ] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [ X] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

File: permissions/plugin-apprenda.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: "apprenda"
+paths:
+- "org/jenkins-ci/plugins/apprenda"
```
Comment (daniel-beck) [inline permissions/plugin-apprenda.yml]: Only correct if https://github.com/jenkinsci/apprenda-plugin/blob/e3781ea022adf8257f1caae3f512795967607877/pom.xml#L30 didn't exist.

Comment (michmike) [reply permissions/plugin-apprenda.yml]: @daniel-beck should i remove line 30 from the BOM then?

Comment (daniel-beck) [reply permissions/plugin-apprenda.yml]: @michmike That, or adapt the YAML file in this PR. Either works.

**Comment by michmike** [comment]
@daniel-beck  i made changes to the POM to match the yml file. thank you!

**Pull request merged**